include/layers/EWLayer.hpp

```diff
@@ -0,0 +1,36 @@
+#pragma once
+#include <algorithm>
+
+#include "layers/FCLayer.hpp"
+
+template <typename T>
+T relu(const T& value) {
+  if (value > T(0)) {
+    return value;
+  }
+  return T(0);
+}
+
+template <typename ValueType>
+class EWLayer : public Layer<ValueType> {
+ public:
+  EWLayer(const Shape& inputShape, ValueType (*unaryFunc)(const ValueType&)) {
+    unaryFunc_ = unaryFunc;
+    this->inputShape_ = inputShape;
+    this->outputShape_ = inputShape;
```

Comment:
```suggestion
    this->outputShape_ = outputShape;
```

Reply:
isn't output shape the same as input shape?

Reply:
Yes, you can accept it as just "shape" in EWLayer constructor args

Reply:
Changed